samples/gatsby-preview/src/index.js

```diff
@@ -68,8 +55,45 @@ class App extends React.Component {
     );
   };
 
-  openPreviewTab = () => {
-    window.open(this.state.previewUrl);
+  openPreviewTab = async () => {
+    const {
+      parameters: { installation, instance },
+      entry
+    } = this.props.sdk;
+    const { previewUrl } = installation;
```

Comment:
I'd normalize this URL right away - you're using normalized version in L85 but not normalized in L68.